content/docs/dvclive/get-started.md

```diff
@@ -75,7 +75,7 @@ with Live() as live:
 Check the [ML Frameworks](/doc/dvclive/api-reference/ml-frameworks) page for
 more details and other supported frameworks.
 
-### Run the script
+## Run the script
```

Comment:
```suggestion
## Run the code
```

Let's try to get rid of references to a script since it suggests that you can't run in a notebook or other interactive environment.